exporters/otlp/include/opentelemetry/exporters/otlp/otlp_http_client.h

```diff
@@ -251,6 +288,9 @@ class OtlpHttpClient
   // The configuration options associated with this HTTP client.
   const OtlpHttpClientOptions options_;
 
+  /** SSL options. */
+  ext::http::client::HttpSslOptions ssl_options_;
```

Comment:
Is is better to let be a member of `OtlpHttpClientOptions`, many fields are duplicated.

Reply:
Indeed, thanks.

Fixed.